app/views/shared/_transcriber_templates.html.erb

```diff
@@ -191,29 +191,29 @@
                 <div class="labels-container">
                   <!-- "What is the date and time of this observation? <b>You can select the date and time by clicking on the clock icon.</b><br>Should follow the format <nobr><code>04 Aug 1874, 01:48 AM</code></nobr>." -->
                     <div class="scribe_input_label date-label">
-                      <%= content_tag "i", "", 
-                        class: "fa fa-info-circle", 
-                        style: "cursor: help;", 
-                        title: strip_tags(t('transcriber.transcription-box-template.form.fields.labels.datetime')), 
-                        data: { 
-                          content: t('transcriber.transcription-box-template.form.fields.help.datetime').html_safe, 
-                          html: true, toggle: "popover", placement: "auto", trigger: "hover", container: "body"
-                        } 
-                      %> <%= t('transcriber.transcription-box-template.form.fields.labels.datetime') %>
+                      <label for="datetime-help"
+                      style="cursor: help;"
+                      aria-hidden="true" 
+                      title="<%=strip_tags(t('transcriber.transcription-box-template.form.fields.labels.datetime'))%>"
+                      data-toggle="popover" 
+                      data-trigger="hover" 
+                      data-content="<%= t('transcriber.transcription-box-template.form.fields.help.datetime').html_safe %>" 
+                      data-container="body" 
+                      data-html=true
```

Comment:
I think that the value for `data-html` should be `"true"` instead of `true`, so `data-html="true"`. This might solve the problem that you mentioned of the body text in the popover not rendering the HTML correctly.

Reply:
I think the issue is the "<nobr>" tag. When I remove it the text in the "<code>" tag is rendered correctly, the only problem is that it wraps to two lines. Like this:

<img width="1438" alt="Screen Shot 2020-06-23 at 8 57 56 AM" src="https://user-images.githubusercontent.com/43585486/85407697-848bc980-b531-11ea-8839-3bfddcce12ef.png">

I've read that the nobr tag is obsolete: https://developer.mozilla.org/en-US/docs/Web/HTML/Element/nobr

The popover doesn't allow for inline styling but I can try to find another solution if the wrapping is not okay.